src/libcore/ops.rs

```diff
@@ -102,6 +102,9 @@ pub trait Drop {
     ///
     /// After this function is over, the memory of `self` will be deallocated.
     ///
+    /// This function cannot be called explicitly. This is compiler error
+    /// [0040](https://doc.rust-lang.org/error-index.html#E0040).
```

Comment:
Should mention that there’s a `drop<T>(t: T)` function in prelude which will call it for you.